.github/workflows/srsim-tests.yml

```diff
@@ -29,6 +32,13 @@ jobs:
       - name: Move containerlab to usr/bin
         run: sudo mv ./containerlab /usr/bin/containerlab && sudo chown root:root /usr/bin/containerlab && sudo chmod 4755 /usr/bin/containerlab
 
+      - name: Create license file from env vars
+        run: |
+          echo $SRSIM_TEST > ./tests/13-srsim/sros_test.txt
+          echo $SRSIM_LICENSE > ./tests/13-srsim/sros_license.txt
```

Comment:
to adhere to the examples, lets' save the license to `/opt/nokia/sros/license-sros25.txt`